src/api/hooks/useParachainsOverview.ts

```diff
@@ -0,0 +1,92 @@
+import {gql, useQuery} from '@apollo/client';
+import {
+  SubstrateChainAccountInfo,
+  SubstrateChainParachain,
+  SubstrateChainParachainsInfo,
+} from 'src/generated/litentryGraphQLTypes';
+
+export type ParachainsInfo = SubstrateChainParachainsInfo;
+
+export type Parachain = SubstrateChainParachain;
+
+export type AccountInfo = SubstrateChainAccountInfo;
+
+export const PARACHAINS_INFO_QUERY = gql`
+  query getParachainsInfo {
+    substrateChainParachainsInfo {
+      parachainsCount
+      parathreadsCount
+      proposalsCount
+      leasePeriod {
+        currentLease
+        totalPeriod
+        progressPercent
+        remainder
+      }
+    }
+  }
+`;
+
+export const PARACHAINS_QUERY = gql`
+  query getParachains {
+    substrateChainParachains {
+      id
+      name
+      lease {
+        period
+        blockTime
+      }
+      lifecycle
+      lastIncludedBlock
+      lastBackedBlock
+      homepage
+      validators {
```

Comment:
Only query what will be used in the overview screen. Validators, nonVoters, etc, are not used in the list.